### PR TITLE
phpMyAdmin: update to 5.0.4, fix license

### PR DESCRIPTION
**Pull request by kawaiiamber**
Closes [this](https://github.com/void-linux/void-packages/issues/27798).

### CODE REVIEW

File: srcpkgs/phpMyAdmin/template [hunk 1]
```diff
@@ -1,16 +1,16 @@
 # Template file for 'phpMyAdmin'
 pkgname=phpMyAdmin
-version=5.0.2
+version=5.0.4
 revision=2
```
Comment (abenson) [inline srcpkgs/phpMyAdmin/template:4]: Reset `revision` to 1

**Comment by abenson** [comment]
change commit message to `phpMyAdmin: update to 5.0.4, fix license`

File: srcpkgs/phpMyAdmin/template (hunk 1 above)
Comment (kawaiiamber) [reply srcpkgs/phpMyAdmin/template]: Done

**Comment by kawaiiamber** [comment]
> change commit message to `phpMyAdmin: update to 5.0.4, fix license`

Done

**Pull request merged**